src/requirements.txt

```diff
@@ -1,7 +1,9 @@
+black>=20.8b1
```

Comment:
Add `black` and `flake8` to the requirements so that they can be run via `npm run black` and `npm run flake8`, and/or in a git hook.